File: core/core-enum/src/main/java/io/say/better/core/enums/RoleType.java
package io.say.better.core.enums;

import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@RequiredArgsConstructor
public enum RoleType {
	EDUCATOR("교육자"),
	LEARNER("학습자"),
	ADMIN("관리자"),
	NONE("초기 상태");

	private final String description;
}

File: core/core-common/src/main/kotlin/io/say/better/core/common/RoleType.kt
package io.say.better.core.common

enum class RoleType(
    val description: String,
) {
    EDUCATOR("교육자"),
    LEARNER("학습자"),
    EDUCATOR_LEARNER("교육자/학습자"),
    ADMIN("관리자"),
    NONE("초기 상태"),
}
